src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1519,4 +1519,13 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="UseAssignableTypeForForeachVariableDescription" xml:space="preserve">
+    <value>There is no implicit conversion between collection element and iteratoi variable, this will result in runtime exception.</value>
+  </data>
+  <data name="UseAssignableTypeForForeachVariableMessage" xml:space="preserve">
+    <value>Cannot convert type '{0}' to '{1}'</value>
```

Comment:
What about: `Cannot find any implicit conversion between type '{0}' and '{1}'`?

Reply:
Updated to `Cannot implicitly convert type 'type1' to 'type2'.` like in [CS0266](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0266)